CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Added
 
+- Added [`bundle build`](https://github.com/operator-framework/operator-sdk/blob/master/doc/cli/operator-sdk_alpha_bundle_build.md) and [`bundle generate`](https://github.com/operator-framework/operator-sdk/blob/master/doc/cli/operator-sdk_alpha_bundle_generate.md) (under the `alpha` subcommand) which build and generate metadata for [operator bundle images](https://github.com/kevinrizza/enhancements/blob/ec2cf96/enhancements/olm/operator-registry.md), respectively. ([#2076](https://github.com/operator-framework/operator-sdk/pull/2076))
+
```

Comment:
Could you please replace the fork  URL:

> [operator bundle images](https://github.com/kevinrizza/enhancements/blob/ec2cf96/enhancements/olm/operator-registry.md)

IHMO: We should not add ref to ours GitHub accounts.